addon/templates/components/cfb-form-editor/question.hbs

```diff
@@ -120,6 +120,17 @@
         }}
       {{/if}}
 
+      {{#if (and slug (includes f.model.__typename (array
+        "TextQuestion" "TextareaQuestion" "IntegerQuestion" "FloatQuestion"
+        "ChoiceQuestion" "MultipleChoiceQuestion" "DateQuestion" "TableQuestion"
+      )))}}
```

Comment:
I think excluding `FormQuestion` and `StaticQuestion` would be more intuitive

Reply:
Excluding requires that you have the full list present in your mind to know the final list.

Your list is a good example as it is missing the `DynamicChoiceQuestion` and `DynamicMultipleChoiceQuestion` ;-)

Reply:
Fair enough